initializr/src/main/resources/templates/home.html

```diff
@@ -11,174 +11,167 @@
-                    <div class="col-md-8">
-                        <select class="form-control" name="javaVersion" id="javaVersion">
-                            <% javaVersions.content.each { %>
-                            <option value="${it.id}" ${it.default==true ? ' selected' : ''}>${it.name}</option>
+                    <div class="form-group">
+                        <label for="packaging" class="control-label">${packagings.title}</label>
+                        <select class="form-control" id="packaging" name="packaging">
+                            <% packagings.content.each { %>
+                            <option value="${it.id}" ${it.default==true ?
+                            ' selected' : ''}>${it.name}</option>
                             <% } %>
                         </select>
                     </div>
-                </div>
-                <div class="form-group">
-                    <label for="language" class="col-md-3 control-label">${languages.title}</label>
-                    <div class="col-md-8">
+                    <div class="form-group">
+                        <label for="language" class="control-label">${languages.title}</label>
                         <select class="form-control" name="language" id="language">
                             <% languages.content.each { %>
-                            <option value="${it.id}" ${it.default==true ? ' selected' : ''}>${it.name}</option>
+                            <option value="${it.id}" ${it.default==true ?
+                            ' selected' : ''}>${it.name}</option>
                             <% } %>
                         </select>
                     </div>
                 </div>
-                <div class="form-group">
-                    <label for="bootVersion" class="col-md-3 control-label">${bootVersions.title}</label>
-                    <div class="col-md-8">
-                        <select class="form-control" name="bootVersion" id="bootVersion">
-                            <% bootVersions.content.each { %>
-                            <option value="${it.id}" ${it.default==true ? ' selected' : ''}>${it.name}</option>
-                            <% } %>
-                        </select>
+            </div>
+            <div class="row">
+                <h2>Quick add dependencies</h2>
+
+                <p>Add Spring Boot Starters and dependencies to your application</p>
+
+                <div class="form-group col-md-6 col-sm-12">
+                    <div class="form-group">
+                        <input id="autocomplete" class="form-control" type="text" placeholder="Search for Starters"
+                               name="autocomplete">
+                    </div>
+                    <div class="message">
+                        <div class="alert alert-warning invisible" role="alert">This dependency is not available for the
```

Comment:
For now, the autocompletion engine will suggest dependencies even if they're not available for the currently chosen Boot version. In case developers try to add those, this warning message is displayed for 5 seconds.

There's a way to dynamically filter this, but this rules out prefetching the `dependencies.json` information and all autocomplete calls must be sent to the server (i.e. search and filtering on the server side)...